libs/application/templates/driving-license/src/lib/utils/formUtils.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { createApplication } from '@island.is/testing/fixtures'
+import { NO, YES } from '../constants'
+import {
+  isVisible,
+  isApplicationForCondition,
+  needsHealthCertificateCondition,
+  allowFakeCondition,
+  hasCompletedPrerequisitesStep,
+} from './formUtils'
+
+describe('isVisible', () => {
+  it('returns true when all functions return true', () => {
+    expect(
+      isVisible(
+        () => true,
+        () => true,
```

Comment:
is () => true, supposed to come twice?

Reply:
yeah, this isVisible function should return true when all functions return true